engines/hpl1/console.cpp

```diff
@@ -0,0 +1,38 @@
+/* ScummVM - Graphic Adventure Engine
```

Comment:
I think it is safe to remove this file. It has no actual content anyway besides the boilerplate code.